### ISSUE TITLE
Pasting a Table - Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by chrisvanderkooi** [froala/wysiwyg-editor#4198]
##### Expected behavior.
When pasting a table from another editor like Google Docs, it should paste the table as a table in Froala and allow editing of table like when you create a new table inside the Froala editor.

##### Actual behavior.
Pasting a table from Google Docs works and displays as a table, but when clicking into a cell, a javascript error occurs and now table cell popup is displayed. This occurs using the angular-froala-wysiwyg npm package as well as the demo on the Froala Editor main page.

##### Steps to reproduce the problem.
Create a new Google Docs document. Create a table, insert some content into a few cells. Copy the document contents and paste into a Froala Editor instance. The table displays, but as soon as you click into a cell to edit the table cell, a console.error is show in the console and no table cell edit popup displays.

##### Editor version.
3.2.6

##### OS.
macOS Mojave 10.14.6

##### Browser.
Chrome 89

##### Screenshot.
![image](https://user-images.githubusercontent.com/819171/112040914-63312b80-8b0b-11eb-9338-8699995b964b.png)

```
Uncaught TypeError: Cannot read property 'length' of undefined
    at _currentSelection (table.js?version=1.0.1:1715)
    at _selectionOffset (table.js?version=1.0.1:1817)
    at _showEditPopup (table.js?version=1.0.1:92)
    at HTMLTableCellElement._mouseUp (table.js?version=1.0.1:2030)
    at froala_editor.pkgd.min.js?version=1.0.1:7
    ```

**Comment by earshinov** [comment]
Duplicate of #4155 and #4191

**Comment by AkshayCM** [comment]
Thank you for your feedback.
This issue is not reproducible on v4.0.9 branch.

**Comment by MeghashreeAP** [comment]
Thank you for your feedback.
This issue is not reproducible on latest branch.

**Comment by earshinov** [comment]
I can confirm that this issue is fixed in 4.0.9

**Comment by chrisvanderkooi** [comment]
Can also confirm that it's fixed in 4.0.9. With that said, I'll close the issue.
